src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  area: number;
 }
 
 export class Triangle {
+  shape: Shape;
+
+  area: number;
```

Comment:
Remove the area property from the Triangle class, since it should be defined in the Figure interface.